python/lib/scheduler/dmod/test/it_RedisBackedJobManager.py

```diff
@@ -13,7 +13,18 @@
 
 # TODO: also add another test for the job manager factory and its factory method
 
-
+try:
+    import pytest
+    import os
+    # https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables
+    in_ci = os.environ.get("CI", False) == "true"
```

Comment:
Curious if the use of `False` versus `"false"` was deliberate (following some convention) or unintentional.  It's fine, I just spent an extra couple of minutes thinking about whether it could lead to any trouble.

Reply:
My thinking was the only truthy case should be when the string `true` is set. In all other cases it should be `False`. Thoughts?

Reply:
It's mostly just counterintuitive to me at first glance.  I'd probably have used "false" as the default value for the `get` call and compared a string to a string.  But this is perfectly correct.